src/popup/snooze.html

```diff
@@ -6,7 +6,21 @@
   </head>
   <body>
     <div id="app">
-      <div class="panel-wrapper"><div><div id="main" class="static panel active"><ul class="times"><li class="option" id="later"><img src="../icons/later_today.svg" class="icon"><div class="title">Later Today</div><div class="date">...</div></li><li class="option" id="tomorrow"><img src="../icons/tomorrow.svg" class="icon"><div class="title">Tomorrow</div><div class="date">...</div></li><li class="option" id="weekend"><img src="../icons/weekends.svg" class="icon"><div class="title">This Weekend</div><div class="date">...</div></li><li class="option" id="week"><img src="../icons/next_week.svg" class="icon"><div class="title">Next Week</div><div class="date">...</div></li><li class="option" id="month"><img src="../icons/next_month.svg" class="icon"><div class="title">Next Month</div><div class="date">...</div></li><li class="option" id="next"><img src="../icons/next_open.svg" class="icon"><div class="title">Next Open</div><div class="date"></div></li><li class="option" id="pick"><img src="../icons/pick_date.svg" class="icon"><div class="title">Pick a Date/Time</div><div class="date"></div></li></ul><div class="footer"><div class="manage"><span>Manage Snoozed Tabs</span></div></div></div></div>
+      <div class="panel-wrapper">
```

Comment:
I wonder why it was all on one line previously… Perhaps something about extra whitespace messing with the styling? Could you add a comment with screenshots from before and after this patch is applied?

Reply:
Removed unneeded change.